utils/python/tests/scripts_regression_tests.py

```diff
@@ -27,6 +27,7 @@
 FAST_ONLY   = False
 NO_BATCH    = False
 NO_CMAKE    = False
+TEST_ROOT = None
```

Comment:
I've add an option to scripts_regression_tests --test-root, if the option is not provided one is generated and all test output is written into subdirectories of this directory.